aziotctl/src/internal/check/checks/daemons_running.rs

```diff
@@ -117,7 +117,7 @@ impl Checker for DaemonRunningTpmd {
         };
 
         if !using_tpmd {
-            return CheckResult::Skipped;
+            return CheckResult::Ignored;
```

Comment:
Not related to this PR, but I am curious why the TPM service status is being checked if the provisioning information is missing in either current or previous configuration. Should we bother checking tpmd status if `aziot init` was not run?

Reply:
See discussion in https://github.com/Azure/iot-identity-service/pull/97